client/my-sites/upgrades/domain-management/domain-connect/index.jsx

```diff
@@ -10,4 +10,9 @@ export default ( router ) => {
 		domainConnectAuthorize,
 		makeLayout
 	);
+	router(
+		'/*',
```

Comment:
We could use /domain-connect/* since that is what is routed here, but this is just meant to be a catch-all that will handle _everything_ that wasn't already handled. I was just trying to be as general as possible.